### PR TITLE
Create VideoTutorials.rst

### PR DESCRIPTION
**Pull request by chrisvo1992**
Page with resourceful video tutorials

### CODE REVIEW

File: docs/reST/tut/VideoTutorials.rst [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+.. title:: Useful Video Tutorials
+
+
+Useful Video Tutorials
+========================
+Pygame is fun and easy to learn but for some developers, video tutorials are the best way to learn.
+The purpose of this page is to pr ovide links to the most popular and resourceful videos
+for popular topics. Watch videos to learn or to get a refresher on useful topics. 
+
+Total Beginners guide to Pygame
```
Comment (novialriptide) [inline docs/reST/tut/VideoTutorials.rst:18]: Rename to "Total Beginners Guide to Pygame"

File: docs/reST/tut/VideoTutorials.rst [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+-----------------------
+https://youtu.be/FfWpgLFMl7w
```
Comment (novialriptide) [inline docs/reST/tut/VideoTutorials.rst:20]: This video doesn't exist.

**Review by novialriptide — CHANGES_REQUESTED** [review]
I'm not particularly sure about this since most of these videos seem very randomly grabbed from the internet. Maybe we should attach Youtuber's playlist of pyame tutorials instead?

File: docs/reST/tut/VideoTutorials.rst [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+ Useful Video Tutorials
+**************************
+
+.. title:: Useful Video Tutorials
+
+
+Useful Video Tutorials
+========================
+Pygame is fun and easy to learn but for some developers, video tutorials are the best way to learn.
+The purpose of this page is to pr ovide links to the most popular and resourceful videos
```
Comment (Starbuck5) [inline docs/reST/tut/VideoTutorials.rst:15]: "pr ovide" -> "provide"

**Comment by Starbuck5** [comment]
This has been stale for a while without reply by OP, so I'm going to go ahead and close this.

**Pull request closed without merging**